staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -852,6 +848,23 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	return &apiResource, nil
 }
 
+// getOpenAPISchema builds the openapi schema for a single resource model to be given to each handler. It will
+// return nil if the apiserver doesn't have openapi enabled, or if the specific path should be ignored by openapi.
+func (a *APIInstaller) getOpenAPISchema(rootPath string, kind schema.GroupVersionKind, sampleObject interface{}) (openapiproto.Schema, error) {
+	if a.group.OpenAPIConfig == nil {
+		return nil, nil
+	}
+	pathsToIgnore := openapiutil.NewTrie(a.group.OpenAPIConfig.IgnorePrefixes)
+	if pathsToIgnore.HasPrefix(rootPath) {
```

Comment:
@jennybuckley , I wanted to point out one thing is that this does not exactly match the behavior of 1.10 release.

In 1.10 release, we would use IgnorePrefixes like `/apis/admission.stash.appscode.com/v1alpha1/restics` and that will be enough.

In 1.11 release with this commit, I have to use `/apis/admission.stash.appscode.com/v1alpha1` as IgnorePrefixes. This is because `ws.RootPath()` is `/apis/admission.stash.appscode.com/v1alpha1`.

Here is a pr that we had to do to fix this: https://github.com/appscode/stash/pull/536

Reply:
Hmm it's interesting that this was necessary, I just took ```ws.RootPath()``` from https://github.com/kubernetes/kube-openapi/blob/master/pkg/builder/openapi.go#L200

Reply:
I'll look into that. Thanks

Reply:
The implication is that if someone wants to ignore prefixes for for some of the resources for an apiVersion, it will not be possible.

Reply:
I think this can be fixed by using ```resource``` instead of ```ws.RootPath()```